### ISSUE TITLE
[GREENFOOT-419] Greenfoot.ask() method prevents later keyboard input

### ISSUE THREAD

**Issue by neilccbrown** [k-pet-group/BlueJ-Greenfoot#1763]
In version 2.4.1, the Greenfoot.ask() method works initially, but after it has finished, keyboard focus does not return properly to the Greenfoot window, and thus keyboard input does not work until the scenario is reset.

---

**Issue metadata**

- Issue type: Bug
- Priority: Medium
- Fix versions: 2.4.2
